core/amber/src/main/scala/edu/uci/ics/texera/web/TexeraCompilerWebApplication.scala

```diff
@@ -0,0 +1,85 @@
+package edu.uci.ics.texera.web
+
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
+import com.github.toastshaman.dropwizard.auth.jwt.JwtAuthFilter
+import com.typesafe.scalalogging.LazyLogging
+import edu.uci.ics.amber.engine.common.AmberConfig
+import edu.uci.ics.texera.Utils
+import edu.uci.ics.texera.web.TexeraWebApplication.parseArgs
+import edu.uci.ics.texera.web.auth.JwtAuth.jwtConsumer
+import edu.uci.ics.texera.web.auth.{
+  GuestAuthFilter,
+  SessionUser,
+  UserAuthenticator,
+  UserRoleAuthorizer
+}
+import edu.uci.ics.texera.web.resource.WorkflowCompilationResource
+import io.dropwizard.auth.{AuthDynamicFeature, AuthValueFactoryProvider}
+import io.dropwizard.setup.{Bootstrap, Environment}
+import org.eclipse.jetty.server.session.SessionHandler
+import org.glassfish.jersey.server.filter.RolesAllowedDynamicFeature
+
+object TexeraCompilerWebApplication {
+  def main(args: Array[String]): Unit = {
+    val argMap = parseArgs(args)
+
+    new TexeraCompilerWebApplication().run(
+      "server",
+      Utils.amberHomePath
+        .resolve("src")
+        .resolve("main")
+        .resolve("resources")
+        .resolve("texera-compiler-web-config.yml")
+        .toString
```

Comment:
make it less depend on amber. ideally this component should be stand alone.

Reply:
Here the `amberHomePath` is the directory containing the both community and texera. So I think I still need to use it and put everything under it. Once we have separate the community and texera in separate folders. I can get rid of `amberHomePath`, use `texeraHomePath` or something similar.

Reply:
So my plan is to not change this part in this PR.

Reply:
ok, can be done in future PRs.